src/System.Diagnostics.Process/src/System/Collections/Specialized/StringDictionaryWithComparer.cs

```diff
@@ -0,0 +1,170 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Diagnostics;
+using System;
+using System.Globalization;
+using System.Collections.Generic;
+
+namespace System.Collections.Specialized {
```

Comment:
Nit: The corefx [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) is to put braces on a new line (applies throughout):

> 1) We use Allman style braces, where each brace begins on a new line.